src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDeposit.scala

```diff
@@ -218,31 +218,75 @@ object AddDatasetMetadataToDeposit {
   }
 
   def createSpatialPoints(dataset: Dataset): Seq[Elem] = {
-    // coordinate order latitude (DCX_SPATIAL_Y), longitude (DCX_SPATIAL_X)
-    dataset.rowsWithValuesForAllOf(composedSpatialPointFields).map(mdKeyValues =>
+    dataset.rowsWithValuesForAllOf(composedSpatialPointFields).map(mdKeyValues => {
+      val srsName = createSrsName(mdKeyValues)
+      // coordinate order x, y = longitude (DCX_SPATIAL_X), latitude (DCX_SPATIAL_Y)
+      val xy = s"${mdKeyValues.getOrElse("DCX_SPATIAL_X", "")} ${mdKeyValues.getOrElse("DCX_SPATIAL_Y", "")}"
```

Comment:
put `lazy` before both `xy` and `yx` to make sure only one is computed. Or even better, calculate the `DCX_SPATIAL_X` and `DCX_SPATIAL_Y` separately and do the correct concat in the pattern match on `srsName` below